DESCRIPTION

```diff
@@ -24,4 +24,8 @@ Suggests:
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
 Imports: 
-    ggplot2
\ No newline at end of file
+    tibble,
+    dplyr,
+    knitr,
+    kableExtra
```

Comment:
```suggestion
    kableExtra,
```

Reply:
please check this carefully as I'm not sure if this is supposed to have a comma here or not, I'm just looking at surrounding formatting